desktop-src/dataxchg/html-clipboard-format.md

```diff
@@ -21,244 +21,346 @@ ms.date: 05/31/2018
 
-            StartFragment:130
+Remember, the _selection_ can represent a run of text which can start in any given element and end in **any** subsequent - or ancestor - element. Consequently, it is impossible to represent a text selection using HTML.
 
-            EndFragment:150
 
-            **StartSelection:130**
-
-            **EndSelection:150**
-
-            <!DOCTYPE ...>
+## Scenarios
 
-            &lt;BODY&gt;
+The following scenarios describe how the IE4/MSHTML HTML editor handles HTML cut and paste; other applications may or may not follow these scenarios. The clipboard format described here is intended to allow flexibility for how an application chooses to function. (These scenarios show only good HTML, that is, no overlapping tags.)
 
-            <!-- StartFragment -->>
+### Scenario 1 - Simple Fragment of HTML
 
-            **<B>bold</B><I><B>This is bold italic</B>This</I>**
+#### HTML Text
 
-            <!-- EndFragment -->
+> ```
+> <body>This is normal. <b>This is bold.</b> <i><b>This is bold italic.</b> This is italic.</i></body>
+> ```
 
-            &lt;/BODY&gt;
+This will appear as follows:
 
-            &lt;/HTML&gt;
+> This is normal. **This is bold.**  **_This is bold italic._**  _This is italic._
 
-        -   In this scenario only the BODY tag and the HTML tag appear in the context as it precedes the selected fragment. Note that start tags and end tags are included in the context. The selection, as delimited by StartSelection and EndSelection, is shown in bold.
+<!-- (TODO: Screenshot of Word showing the selected text *before* copying? omg)
+(TODO: Screenshot of Word after pasting it) --> 
 
-2.  Fragment of a table in HTML.
-    -   -   HTML text:
+When your user has loaded the above HTML text into an MSHTML-based application (MSHTML, _aka_ Trident, was Internet Explorer's engine), MSHTML handles copying a substring of HTML as follows:
 
-            &lt;BODY&gt;<TABLE BORDER><TR><TH ROWSPAN=2>Head1</TH><TD>Item 1</TD><TD>Item 2</TD><TD>Item 3</TD><TD>Item 4</TD></TR><TR><TD>Item 5</TD><TD>Item 6</TD><TD>Item 7</TD><TD>Item 8</TD></TR><TR><TH>Head2</TH><TD>Item 9</TD><TD>Item 10</TD><TD>Item 11</TD><TD>Item 12</TD></TR></TABLE>&lt;/BODY&gt;
+1. The user selects a text from the start of the word "bold" (in "This is bold.") through to the end of the word "This" (in "This is italic.") without any leading nor trailing whitespace in the selection,  
+   i.e. The selection is represented by the square-brackets in `"This is normal **This is \[bold**  **_This is bold italic_**  _This\] is italic_"`.
+2. The user clicks the Copy command button to copy the selection to the clipboard.
+3. MSHTML will consequently place this HTML text into the Windows Clipboard:
```

Comment:
```suggestion
3. MSHTML will place this HTML text into the Windows Clipboard as follows:
```

OR 

```suggestion

MSHTML will place this HTML text into the Windows Clipboard as follows:
```


Two susgestions - this is the result, not sure if it should be part of this list or not.